File: data/src/main/java/com/blackmirror/dongda/data/model/response/BaseResponseBean.java
package com.blackmirror.dongda.data.model.response;

import java.io.Serializable;

/**
 * Created by Ruge on 2018-05-04 下午2:02
 */
public class BaseResponseBean implements Serializable {
    public String status;//ok 成功 其他失败

    /**
     * error : {"code":-9005,"message":"token过期"}
     */

    public ErrorBean error;

    public static class ErrorBean implements Serializable{
        /**
         * code : -9005
         * message : token过期
         */

        public int code;
        public String message;//错误信息
    }
}

File: data/src/main/java/com/blackmirror/dongda/data/model/response/BaseResponseBean.kt
package com.blackmirror.dongda.data.model.response

import java.io.Serializable

/**
 * Created by Ruge on 2018-05-04 下午2:02
 */
open class BaseResponseBean : Serializable {
    var status: String? = null//ok 成功 其他失败

    /**
     * error : {"code":-9005,"message":"token过期"}
     */

    var error: ErrorBean? = null

    inner class ErrorBean : Serializable {
        /**
         * code : -9005
         * message : token过期
         */

        var code: Int = 0
        var message: String? = null//错误信息
    }
}
